x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -55,7 +55,7 @@ export default function ({ getService }: FtrProviderContext) {
   const testHistoryIndex = '.kibana_task_manager_test_result';
 
   // Failing: See https://github.com/elastic/kibana/issues/141002
```

Comment:
Can remove this comment too :)

Reply:
Resolved in https://github.com/elastic/kibana/pull/141441/commits/0b62dbeef01c86a650f36711627c7c20e71ef68b